VSRAD.Package/DebugVisualizer/SliceVisualizer/SliceVisualizerTable.cs

```diff
@@ -26,27 +34,62 @@ public SliceVisualizerTable(IFontAndColorProvider fontAndColor) : base()
             AllowUserToResizeRows = false;
             AutoGenerateColumns = false;
             HeatMapMode = false;
+            ColumnStyling = new SliceColumnStyling(this, _context.Options.VisualizerAppearance);
             ColumnHeadersHeightSizeMode = DataGridViewColumnHeadersHeightSizeMode.DisableResizing;
+            MouseClick += ShowContextMenu;
+            CellMouseEnter += DisplayCellStatus;
+            MouseLeave += (s, e) => _context.StatusString = ""; // clear status bar on leaving control
 
             _state = new TableState(this, 60);
 
             _mouseMoveController = new MouseMove.MouseMoveController(this, _state);
             _selectionController = new SelectionController(this);
             _ = new SliceRowStyling(this);
-            _ = new SliceCellStyling(this, _state, fontAndColor);
+            _ = new SliceCellStyling(this, ColumnStyling, fontAndColor, _context);
```

Comment:
I'd add a static method calling the constructor to these buds and would use it instead of the constructor if you want to keep it like this. It's kind of not a C# way to implement things as side effects of constructor and looks weird.

Reply:
Thanks for suggestion, implemented in a9d6a25, did you mean something like this?